nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -547,8 +547,8 @@
       </para>
      </listitem>
     </itemizedlist>
-
-     This also configures the kernel to pass coredumps to <literal>systemd-coredump</literal>.
+     This also configures the kernel to pass coredumps to <literal>systemd-coredump</literal>,
+     and restricts the SysRq key combinations to the sync command only.
      These sysctl snippets can be found in <literal>/etc/sysctl.d/50-*.conf</literal>,
      and overridden via <link linkend="opt-boot.kernel.sysctl">boot.kernel.sysctl</link>
      (which will place the parameters in <literal>/etc/sysctl.d/60-nixos.conf</literal>).
```

Comment:
Maybe we could also explain here how to get the old setting back.  Like the "to stick to the old behavior ... set boot.kernel.sysctl..." sentence right below.

Reply:
We already explain how to override values set in those snippets.

Listing each of those individually, with instructions on how to get to the old value would increase this paragraph quite a bit, and I'd like to keep it terse.

I think it's enough pointing at it, if someone stumbles over it, they'll likely look up the snippet file and think about what values to set anyways.